editoast/database/src/tables.patch

```diff
@@ -0,0 +1,15 @@
+diff --git i/editoast/database/src/tables.rs w/editoast/database/src/tables.rs
+index ce819c9d11..d4945ef4a4 100644
+--- i/editoast/database/src/tables.rs
++++ w/editoast/database/src/tables.rs
+@@ -970,6 +970,10 @@ diesel::allow_tables_to_appear_in_same_query!(
```

Comment:
Wouldn't the raw line numbers need to be updated for each file modification before line 970?

Reply:
Can use `tables_patch.rs`